### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by meirkeller** [glauberportella/password-hash#9]
Can you please make a new release with the `bcrypt` #5 update?

**Comment by daniel-maegerli** [comment]
Any chance to publish a new update, @glauberportella ?

**Comment by glauberportella** [comment]
Hello, sorry about the waiting time. I will investigate the #5 update to understand and return with an update as soon as possible, probably on the next week.

**Comment by edencorbin** [comment]
Yes please if possible I get bcrypt install issues with node 18.

**Comment by marco-bertelli** [comment]
not a solution but a working fix, i use this fix in node 18 and work as before, add the current snippet to your package.json:

```
  "overrides": {
    "node-phpass": {
      "bcrypt": "5.0.0"
    }
  },
```

this will force to use a specififc working version of bcrypt, overriding the libs setting

**Comment by coxato** [comment]
Hi guys, I just made a fork and uploaded it to npm 
https://github.com/coxato/password-hash

I just did 2 things 
1) I merged the branch of this repo (dependabot/npm_and_yarn/crypto-js-4.2.0) to master
2) I updated the package.json to 
```
"dependencies": {
    "bcryptjs": "^3.0.2",
    "crypto-js": "^4.2.0",
    "locutus": "^2.0.32"
 }
```

you can install it with 
`npm install @coxato/node-phpass`

thanks so much @glauberportella for this package 👍
